source/_components/climate.toon.markdown

```diff
@@ -0,0 +1,32 @@
+---
+layout: page
+title: "Toon van Eneco"
+description: "Instructions on how to integrate the Toon thermostat with Home Assistant."
+date: 2017-05-10 12:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: toon.png
+ha_category: Climate
+ha_release: 0.45
+---
+
+
+The `toon` component platform can be used to control your Toon thermostat. This component adds a climate device for your Toon thermostat and sensors for power and gas consumption. The component also auto-detects any smart plugs connected, solar panels and smoke detectors connected to Toon and adds sensors and switches for them.
+
+To use your Toon thermostat in your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+toon:
+    username: YOUR_USERNAME
+    password: YOUR_PASSWORD
+```
+
+- **username** (*Required*): Username for Mijn Eneco.
```

Comment:
Could you add a short part about that? Your comment here covers it with some rephrasing :)